sdks/python/apache_beam/utils/dependency.py

```diff
@@ -116,19 +116,17 @@ def _dependency_file_download(from_url, to_folder):
   return local_download_file
 
 
-def _stage_extra_packages(extra_packages,
-                          staging_location,
-                          file_copy=_dependency_file_copy, temp_dir=None):
+def _stage_extra_packages(extra_packages, staging_location, temp_dir,
+                          file_copy=_dependency_file_copy):
   """Stages a list of local extra packages.
 
   Args:
     extra_packages: Ordered list of local paths to extra packages to be staged.
     staging_location: Staging location for the packages.
+    temp_dir: Temporary folder where the resource building can happen.
```

Comment:
Add a note here that the function is not responsible for cleaning anything that happens in here because the callers are supposed to do that.